src/desktop.md

```diff
@@ -372,43 +414,61 @@ In addition to that executable, you need the following:
 Place the DLL files in a directory next to the executable
 and the other DLLs, and bundle them together in a zip file.
 
+(FIX: Do you need a signing certificate of some kind?)
+
 [deployment example walkthroughs]: https://docs.microsoft.com/en-us/cpp/windows/deployment-examples?view=vs-2019
 
-#### macOS
+### macOS
 
-To distribute your macOS application, you can either [distribute your app via the macOS App Store][], 
-or alternatively you can distribute the `.app` itself, for example on your own website.  
-However, as of macOS 10.14.5 you need to notarize your macOS application before distributing 
+To distribute your macOS application, you can either
+[distribute it through the macOS App Store][], 
+or you can distribute the `.app` itself,
+perhaps from your own website.
+However, as of macOS 10.14.5 you need to notarize
```

Comment:
Since this is going to be the case going forward, maybe just "As of macOS 10.14.5..."?